src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -60,7 +60,15 @@ public static Process Start(string fileName, string arguments, string userName,
         [UnsupportedOSPlatform("tvos")]
         public void Kill()
         {
-            if (OperatingSystem.IsIOS() || OperatingSystem.IsTvOS())
+            if (OperatingSystem.IsMacCatalyst())
+            {
+                // TODO Sandbox detection hasn't been implemented yet (https://github.com/dotnet/runtime/issues/61504)
```

Comment:
What is the error code returned by the OS API when the sandbox is enabled? Would it be better to just call the OS API and check for that error code instead of trying to detect the sandbox upfront?

Reply:
This was resolved with a check in `Interop.libproc.cs`